src/main/java/duke/task/Task.java

```diff
@@ -68,26 +68,25 @@ public static Task fromStorageString(String storageString) {
         Task task;
 
         switch (taskType) {
-            case TODO:
-                task = new Todo(storageStringArray[2]);
-                break;
-            case EVENT:
-                try {
-                    task = new Event(storageStringArray[2], storageStringArray[3]);
-                } catch (DukeException exception) {
-                    throw new RuntimeException(exception.getMessage());
-                }
-                break;
-            case DEADLINE:
-                try {
-                    task = new Deadline(storageStringArray[2], storageStringArray[3]);
-                } catch (DukeException exception) {
-                    throw new RuntimeException(exception.getMessage());
-                }
-
-                break;
-            default:
-                throw new RuntimeException(String.format("Invalid task type %s.", taskType));
+        case TODO:
+            task = new Todo(storageStringArray[2]);
+            break;
+        case EVENT:
+            try {
+                task = new Event(storageStringArray[2], storageStringArray[3]);
+            } catch (DukeException exception) {
+                throw new RuntimeException(exception.getMessage());
+            }
+            break;
+        case DEADLINE:
+            try {
+                task = new Deadline(storageStringArray[2], storageStringArray[3]);
+            } catch (DukeException exception) {
+                throw new RuntimeException(exception.getMessage());
+            }
+            break;
+        default:
+            throw new RuntimeException(String.format("Invalid task type %s.", taskType));
```

Comment:
I like that the indentation for switch cases was followed. I think you took note of the coding standards and followed them strictly.